Untitled-1.html

```diff
@@ -0,0 +1,36 @@
+<DOCTYPE html>
+<html lang="en">
+<head>
+	<link rel="stylesheet" href="arkusz.css">
+	<nav>
```

Comment:
To trafia do header nie head

W head są tylko informacje o stronie i rożne podlączenia, bez treści